packages/react-start/src/plugin/vite.ts

```
@@ -1,11 +1,11 @@
 import {
   START_ENVIRONMENT_NAMES,
   tanStackStartVite,
-} from '@tanstack/start-plugin-core'
+} from '@tanstack/start-plugin-core/vite'
 import type {
   TanStackStartViteInputConfig,
   TanStackStartVitePluginCoreOptions,
-} from '@tanstack/start-plugin-core'
+} from '@tanstack/start-plugin-core/vite'
```

ESLint

[error] `@tanstack/start-plugin-core/vite` type import should occur after import of `./shared`

(import/order)